shared/src/main/scala/StringHelpers.scala

```diff
@@ -54,4 +129,56 @@ object StringHelpers:
       if index == -1 then c
       else mapping((index + 1) % mapping.length)
     }.mkString
+
+  // https://github.com/DennisMitchell/jellylanguage/blob/70c9fd93ab009c05dc396f8cc091f72b212fb188/jelly/interpreter.py#L1055
+  def sss(compressed: String)(using ctx: Context): String =
```

Comment:
Might want to either add a doc comment saying that this is for decompression or rename the method itself to `decompress` or something simlar

Reply:
I'd be fine with renaming it.